### PR TITLE
Update the project readme

### PR DESCRIPTION
**Pull request by fchabouis**
I added some informations about the metadata and the checks severity.
Also modified existing parts to make the documentation more friendly.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,94 +1,293 @@
 # Validate GTFS files
 
-_General Transit Files Specification_ is a file format defining of transit timetables.
+The General Transit Feed Specification (GTFS) defines a common format for public transportation schedules and associated geographic information.
```
Comment (Tristramg) [inline readme.md]: maybe a link to https://gtfs.org ?

File: readme.md [hunk 2]
```diff
@@ -1,94 +1,293 @@
...
 
 ```
 git clone https://github.com/etalab/transport-validator/
 cd transport-validator
 ```
+## Run the validator
 
-3. Run it as standalone
+### Rune file by file
+The development version can be run as:
```
Comment (Tristramg) [inline readme.md]: There isn’t really a reason to run the debug compilation (unless your are developing on the validator).

The name `development` might be misleading (as dev vs. release version)

File: readme.md [hunk 3]
```diff
@@ -1,94 +1,293 @@
...
 2. Clone the project:
 
 ```
 git clone https://github.com/etalab/transport-validator/
 cd transport-validator
 ```
+## Run the validator
 
-3. Run it as standalone
+### Rune file by file
```
Comment (Tristramg) [inline readme.md]: > 
```suggestion
### Run from a local directory
```

**Review by Tristramg — COMMENTED** [review]
:heart_eyes: Thank you!

File: readme.md (hunk 1 above)
Comment (thbar) [inline readme.md]:
```suggestion
The General Transit Feed Specification ([GTFS](https://gtfs.org/)) defines a common format for public transportation schedules and associated geographic information.
```

File: readme.md [hunk 4]
```diff
@@ -1,94 +1,293 @@
...
 ## Online tool
-transport-validator is the tool used by the [French National Access Point](https://transport.data.gouv.fr/) to validate GTFS files. If you want to try it, you can validate your own files at [this address](https://transport.data.gouv.fr/validation?locale=en).
+transport-validator is the tool used by the [French National Access Point](https://transport.data.gouv.fr/) to validate GTFS files. If you want to use it online, you can validate your own files at [this address](https://transport.data.gouv.fr/validation?locale=en).
 
-## List of checks done by the validator
 
-The checks performed by the validator can be seen in the file [issues.rs](https://github.com/etalab/transport-validator/blob/master/src/issues.rs#L21-L83).
+## Validation output
+Validation output is twofold:
+* it gives useful informations about the validated file, under the `metadata` entry
```
Comment (thbar) [inline readme.md]: I believe `information` is uncountable, so I suggest:

```suggestion
* it gives useful information about the validated file, under the `metadata` entry
```

File: readme.md [hunk 5]
```diff
@@ -1,94 +1,293 @@
...
+* it lists a serie of validation items, with a corresponding severity, under the `validations` entry. When relevant, geographical data (geojson) related to the issue is given to ease file debugging.
```
Comment (thbar) [inline readme.md]:
```suggestion
* it lists a serie of validation items, with a corresponding severity, under the `validations` entry. When relevant, geographical data ([GeoJSON](https://geojson.org/)) related to the issue is given to ease file debugging.
```

File: readme.md [hunk 6]
```diff
@@ -1,94 +1,293 @@
...
 
-Here is a human friendly list of them :
+The output is by default formatted in `json`, but `yaml` si also available. See [Options](https://github.com/etalab/transport-validator#options) for more infos.
```
Comment (thbar) [inline readme.md]:
```suggestion
The output is by default formatted in `json`, but `yaml` is also available. See [Options](https://github.com/etalab/transport-validator#options) for more information.
```

File: readme.md [hunk 7]
```diff
@@ -1,94 +1,293 @@
...
+        ...
+    },
+    "validations": {
+       ...
+    }
+}
+```
 
+### Metadata
+Give some useful informations about the validated file content:
```
Comment (thbar) [inline readme.md]:
```suggestion
Give useful information about the validated file content:
```

File: readme.md [hunk 8]
```diff
@@ -1,94 +1,293 @@
...
+                ],
+                "type": "FeatureCollection"
+            }
+        }
+    ]
+}
+```
 
-1. This project is written in Rust. You need to first [install it](https://rustup.rs/) on your machine.
+Another example showing the geojson information for an information about two stops too close:
```
Comment (thbar) [inline readme.md]:
```suggestion
Another example showing the GeoJSON information for an information about two stops too close:
```

Duplicate `information`?

**Review by thbar — COMMENTED** [review]
Thanks @fchabouis for taking the time to update the readme!

I have made some small suggestions.

**Comment by fchabouis** [comment]
thanks @Tristramg and @thbar for reviewing. I have made the proposed changes .

**Review by thbar — APPROVED** [review]
Merci!

**Pull request merged**